cmd/utask/root.go

```diff
@@ -90,7 +90,7 @@ func init() {
 
 	viper.BindPFlag(envInit, rootCmd.Flags().Lookup("init-path"))
 	viper.BindPFlag(envPlugins, rootCmd.Flags().Lookup("plugins-path"))
-	viper.BindPFlag(envTemplates, rootCmd.Flags().Lookup("templates-path"))
+	viper.BindPFlag(envTemplates, rootCmd.Flags().Lookup("templates-paths"))
```

Comment:
"templates-paths" here, "templates-path" line 82. I'm not sure we want to change the flag name as in UNIX the "PATH" variable is just a list of paths, we could have the same behavior here

Reply:
agreed